tensorflow/contrib/seq2seq/python/kernel_tests/loss_test.py

```diff
@@ -20,14 +20,58 @@
 from __future__ import print_function
 # pylint: enable=unused-import
 
+import numpy as np
 import tensorflow as tf
 
-
 class LossTest(tf.test.TestCase):
 
-  def testLoss(self):
-    pass
+  def testSequenceLoss(self):
+    with self.test_session() as sess:
+      with tf.variable_scope("root", initializer=
```

Comment:
Please, do not separate lines on "=" like here (i.e., move the whole "initializer=" down).